tutorials/ui/bbcode_in_richtextlabel.rst

```diff
@@ -352,6 +353,14 @@ Reference
       | ``{text}``
       | ``[/opentype_features]``
 
+  * - | **lang**
+      | Overrides the language for ``{text}`` that is set by the **BiDi > Language** property in RichTextLabel.
+        ``{code}`` must be an ISO :ref:`language code <doc_locales>`.
+        This can be used to enforce the use of a specific script for a language without starting a new paragraph.
+        Some font files may contain script-specific substitutes, in which case they will be used.
```

Comment:
You should probably wrap lines to the same width as the rest of the table uses (same above, here it's just heavier).